### ISSUE TITLE
VictoryArea: Element type is invalid ... got undefined

### ISSUE THREAD

**Issue by brianlow** [FormidableLabs/victory-native#115]
Having trouble with VictoryArea (VictoryBar and VictoryLine work)

```js
      <View style={{ flex: 1 }}>
        <VictoryChart>
          <VictoryArea />
        </VictoryChart>
      </View>
```

![image](https://user-images.githubusercontent.com/938138/27990411-7b6a9c68-6413-11e7-90f1-f11aa7725723.png)

Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in.

```bash
$ npm list react-native react-native-svg victory-chart
├── react-native@0.45.1
├── react-native-svg@5.2.0
└─┬ victory-native@0.12.0
  └── victory-chart@21.2.3
```

**Comment by Paul-Todd** [comment]
Same here - any suggestions?

**Comment by chrisbolin** [comment]
@Paul-Todd could you post your code too? is it also happening just with VictoryArea?

**Comment by kaareal** [comment]
I think this is fixed on master.
https://github.com/FormidableLabs/victory-native/blob/master/lib/components/victory-primitives/area.js#L36

It should `return null`, which it does on master but the latest release (v0.12.0) `return []`;
https://github.com/FormidableLabs/victory-native/blob/0f1fe241067ac971b832b95c5ba77968bad659a3/lib/components/victory-primitives/area.js#L36

**Comment by chrisbolin** [comment]
@kaareal thanks for investigating! I believe that bugfix you referred shouldn't affect this situation. 

@Paul-Todd + @brianlow I can't seem to reproduce this. Same version of victory-native, 0.12.0...

<img width="514" alt="screen shot 2017-07-13 at 12 31 57 pm" src="https://user-images.githubusercontent.com/2822048/28164469-6f5bd204-67c7-11e7-9e19-40d4719db8a6.png">

**Comment by blackxored** [comment]
I can confirm that I also had this bug and that switching to `master` fixed it. Can we make a release with this? Don't want to forget I'm depending on `master` ¯\_(ツ)_/

**Comment by chrisbolin** [comment]
@blackxored @kaareal glad to hear that cleared it up. i'll do a victory-native release this week

**Comment by chrisbolin** [comment]
closed with todays release of `victory-native@0.13.0`. please open this issue back up if you still see the issue after moving to the new version.

I'd highly recommend a complete reinstall of your dependencies; my gut is that this issue might have been a dep conflict. (As I wasn't able to see the behavior with 0.12.0)
